src/lib/y2storage/disk_analyzer.rb

```diff
@@ -210,6 +210,10 @@ def windows_partition?(partition)
 
       log.info("#{partition.name} is a windows partition") if is_win
       is_win
+
+    rescue Storage::Exception
+      log.info("#{partition.name} content info cannot be detected")
```

Comment:
Not sure if this should be `log.info` or something more serious like `log.warn`.

Reply:
Let me to change it from `.info` to `.warn`.